pkg/crc/constants/constants.go

```diff
@@ -34,6 +34,7 @@ var (
 	MachineCertsDir    = filepath.Join(MachineBaseDir, "certs")
 	MachineCacheDir    = filepath.Join(MachineBaseDir, "cache")
 	OcCacheDir         = filepath.Join(MachineCacheDir, "oc")
+	HyperKitCacheDir   = filepath.Join(MachineCacheDir, "hyperkit")
```

Comment:
? What is this used for ?

If this is for the driver, I would rather see a generic directory ... `drivers`, but thi is not really 'cache'

Reply:
It's something which we can download from the net when needed, so it's some form of local binary cache. ACK for moving it some place else. `~/.crc/bin` ?

Reply:
Just remembered that I only did the same as what is done for OcCacheDir, the `oc` binary goes there. So maybe keep it for consistency, but open a bug to get this fixed in both cases.

Reply:
It turns out this constant was not even used ;) However there were similar variables elsewhere. Latest iteration of the code puts these in ~/.crc/bin/